open_spiel/spiel.h

```diff
@@ -757,6 +757,30 @@ class Game : public std::enable_shared_from_this<Game> {
   // of chance nodes are not included in this length.
   virtual int MaxGameLength() const = 0;
 
+  // The maximum number of chance nodes occurring in any history of the game.
+  // This is typically something like the number of times dice are rolled.
+  virtual int MaxChanceNodesInHistory() const = 0;
```

Comment:
Can you add a default implementation here rather than making pure virtual. It can simply call SpielFatalError. I think Ed will have another suggestion for this call point as well.